app/components/avo/views/resource_index_component.html.erb

```diff
@@ -63,7 +63,7 @@
       <% if view_type.to_sym == :table %>
         <% if @models.present? %>
           <div class="mt-4">
-            <%= render Avo::PaginatorComponent.new pagy: @pagy, turbo_frame: @turbo_frame || 'none', index_params: @index_params, resource: @resource, parent_model: @parent_model %>
+            <%= render Avo::PaginatorComponent.new pagy: @pagy, turbo_frame: @turbo_frame || 'none', index_params: @index_params, resource: @resource, parent_model: @parent_model, field: field %>
```

Comment:
Would it be a bit cleaner to pass `discreet_pagination` to the component?
The pagination component shouldn't really know about the `@field`
```suggestion
            <%= render Avo::PaginatorComponent.new pagy: @pagy, turbo_frame: @turbo_frame || 'none', index_params: @index_params, resource: @resource, parent_model: @parent_model, discreet_pagination: true/false %>
```

Reply:
make sense, already changed it